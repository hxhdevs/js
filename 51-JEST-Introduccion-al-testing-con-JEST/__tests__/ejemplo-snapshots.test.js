const cliente = {
    nombre: 'Elih Freecs',
    balance:500,
    tipo: 'Premium'
};

describe('Testing al cliente',() => {
    test('Es Elih? ', () => {
      expect(cliente).toMatchSnapshot();
    });
});

// podemos actualizar el snapshot desde terminal confirm
// npm t -- -u 

//se podria modificar desde el mismo snapshot pero lo recomendable es hacerlo desde JEST en terminal