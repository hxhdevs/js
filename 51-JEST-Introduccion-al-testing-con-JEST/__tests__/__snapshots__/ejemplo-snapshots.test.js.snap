// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Elih?  1`] = `
{
  "balance": 500,
  "nombre": "Elih Freecs",
  "tipo": "Premium",
}
`;
